timeline/views.py

```diff
@@ -53,6 +53,19 @@ def get_queryset(self):
         return ProjectEvent.objects.filter(**set_kwargs).order_by('-pk')
 
 
+class EventAdminTimelineMixin:
```

Comment:
If this is only used in `AdminTimelineView`, the methods can go directly under that view class.

Reply:
- [x] Moved